docs/design/Ground_station.tex

```diff
@@ -0,0 +1,72 @@
+\textbf{Autopilot and ground station}
+
+\subsection{Overview}
+
+In order for the multirotor to be controllable by a human being with minimal training, much of the flight systems need to be automated. We have decided for our project to use an open source autopilot called ‘Ardupilot’. The autopilot system consists of several hardware components and 2 software components which are split between the multirotor and the ground station.
+
+\underline{Multirotor hardware}\
+
+\begin{enumerate}
+
+    \item Autopilot flight computer: The autopilot flight computer is the primary hardware component of the autopilot system on the multirotor. In the Ardupilot literature this device is referred to only as the ‘Autopilot’ but to avoid confusion with the broader autopilot system we will be referring to it as the ‘Autopilot flight computer’. The specific board that we are using is the APM 2.6. This board an older model and will thus be using legacy code but will have all of the required functionality.
+  
+  \item GPS and Compass: The GPS and compass will allow the ground station to track the multirotor while it’s flying. Also enables the use of the automatic flight modes.
+ 
+ \item Telemetry module: Radio transceiver that allows for communication with the ground.
```

Comment:
Note: our set up don't have a telemetry module
Note: we should clarify that this is different that the radio transmission that controls the drone (and the autopilot). Telemetry is typically used for information like GPS tags, attitude and altitude indicators, etc.

Reply:
I know, but on the ardupilot website it was referred to as the telemetry module.